packages/wallets/src/components/Dropzone/Dropzone.scss

```diff
@@ -0,0 +1,89 @@
+.wallets-dropzone {
+    display: flex;
+    padding: 2.8rem 15.2rem 3.5rem;
+    justify-content: center;
+    align-items: center;
+    border-radius: 8px;
+    height: 100%;
+    width: 100%;
+    border: 0.1rem dashed var(--system-light-5-active-background, #d6dadb);
+
+    @include mobile {
+        padding: 2.4rem 2.5rem 0.4rem;
+    }
+
+    &--hover,
+    &--active {
+        background-color: var(--general-section-1, #f2f3f4);
```

Comment:
Are we still using hard-coded color codes?

Reply:
We dont have the variables for sass yet, @adrienne-deriv said he will create a codemod for it 👀